doc/substitutions.inc

```diff
@@ -0,0 +1,5 @@
+
+.. _The XMOS XS3 Architecture: https://www.xmos.com/download/The-XMOS-XS3-Architecture.pdf
+.. |newpage| raw:: pdf
```

Comment:
This looks like a multi-line substitution.  Have I interpreted it correctly and does it actually work?

Reply:
The substitution was not working, reading online we need some special packages such as rst2pdf to make it work. 
I removed all the "newpage" substitutions, as I couldn't make it work even without substitutions.

In [LSM-120]( https://xmosjira.atlassian.net/browse/LSM-120) we need to rewrite the documentation, so the page breaks will change too.